storage/innobase/xtrabackup/test/t/skip_orphan_table.sh

```diff
@@ -0,0 +1,27 @@
+. inc/common.sh
+
+start_server
+
+vlog "Case#1 backup with  orphan ibd in datadir "
+cp inc/skip_orpah_ibds/FTS_0000000000000123_BEING_DELETED.ibd  $mysql_datadir/test
```

Comment:
no need to have this inc directory. Just create a table in the server. cp this file to a temp location. drop the tablespace/table. copy back the saved file.

Now take backup..

Reply:
if you need IBD without SDI, keep this as is.